docs_src/security/secrets-config-proxy.md

```diff
@@ -18,9 +18,9 @@ Proxy configuration commands (listed below) require access to the secret store m
 
 # OPTIONS
 
-  * **--configDir** _/path/to/directory/with/configuration.toml_ (optional)
+  * **--configDir** _/path/to/directory/with/configuration.yaml_ (optional)
 
-    Points to directory containing a configuration.toml file.
+    Points to directory containing a configuration.yaml file.
```

Comment:
I just made these changes today :-) Please rebase.

Reply:
As long as you acknowledge that I was first.  Rebased.